spec/acceptance/rest/client_spec.rb

```diff
@@ -963,16 +963,17 @@ def encode64(text)
     context 'request_id generation' do
       context 'Timeout error' do
         context 'with request_id' do
-      let(:client_options) { default_options.merge(key: api_key, http_request_timeout: 0, add_request_ids: true) }
-      it 'includes request_id in ConnectionTimeout error' do
-        begin
-          client.stats
-        rescue Ably::Exceptions::ConnectionTimeout => err
-          expect(err.request_id).to_not eql(nil)
+          let(:client_options) { default_options.merge(key: api_key, http_request_timeout: 0, add_request_ids: true) }
+          it 'includes request_id in ConnectionTimeout error' do
+            begin
+              client.stats
+            rescue Ably::Exceptions::ConnectionTimeout => err
+              expect(err.request_id).to_not eql(nil)
+            end
         end
       end
-    end
-        context 'without request_id' do
+
+      context 'without request_id' do
```

Comment:
This doesn't fix the issue?  @funkyboy what is happening here?  Can you explain what the problem is as this is the second attempt to fix indentation issues and it's failed.  What IDE are you using?

![capto_capture 2018-04-18_06-24-58_pm](https://user-images.githubusercontent.com/43789/38947880-f9ced374-4335-11e8-9549-d03bdba4613d.png)

Reply:
I am using VS code but clearly there’s something wrong going on. Will open an issue to add rubocop, so we don’t waste time on this.